core/controllers/feedback.py

```diff
@@ -282,3 +282,29 @@ def post(self):
         email_manager.send_suggestion_email(
             exploration.title, exploration.id, suggestion.author_id,
             exploration_rights.owner_ids)
+
+
+class InstantFeedbackEmailHandler(base.BaseHandler):
+    """Handles task of sending feedback message emails instantsly."""
+
+    def post(self):
+        payload = json.loads(self.request.body)
+        user_id = payload['user_id']
+        reference = payload['reference']
+        message = feedback_services.get_message(
+            reference['exploration_id'], reference['thread_id'],
+            reference['message_id'])
+        exploration = exp_services.get_exploration_by_id(
+            reference['exploration_id'])
+        thread = feedback_services.get_thread(
+            reference['exploration_id'], reference['thread_id'])
+
+        if 'updated_status' in payload:
```

Comment:
Well sending immediate emails through taskqueue does not actually makes that much sense but in future we can add countdown to it. Thus it is just may be in future if we change from immediate to something other we will not require to recode it. Also taskqueue is specially designed for handling background tasks efficiently and email is prime example of background tasks. 
I think if we send emails directly, without using taskqueue as a medium, it will increase load on the server. Suppose we have 30-40 feedbacks per minute then this much mail calls simultaneously might degrade performance of foreground tasks. Whereas taskqueue excutes tasks sequentially and in efficient manner so that resultant load is does not degrade any significant performance. I think we should use taskqueue here.

I will redesign the logic as you suggested in 1st point.
Thanks.